From 75bd18266d6f7cbe676fd0123c8b8d2cb059361b Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Fri, 27 May 2022 16:38:14 +0200
Subject: [PATCH] Use /data as platform storage location

---
 src/platform/Linux/BUILD.gn | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/platform/Linux/BUILD.gn b/src/platform/Linux/BUILD.gn
index e83f7f714d..ba5f293da4 100644
--- a/src/platform/Linux/BUILD.gn
+++ b/src/platform/Linux/BUILD.gn
@@ -41,6 +41,12 @@ if (chip_mdns == "platform") {
 }
 
 static_library("Linux") {
+  defines = [
+    "FATCONFDIR=\"/data\"",
+    "SYSCONFDIR=\"/data\"",
+    "LOCALSTATEDIR=\"/data\"",
+  ]
+
   sources = [
     "../DeviceSafeQueue.cpp",
     "../DeviceSafeQueue.h",
-- 
2.51.2

